README.rst

```diff
@@ -71,7 +73,13 @@ Then start the Python extension:
 
 .. code-block:: none
 
-  python ./my_table_plugin.py --socket /Users/USERNAME/.osquery/shell.em
```

Comment:
While the `#!/usr/bin/env python` should remove the need to have our explicit `python` invocation here, I'd prefer if we left this in as these docs should hold for Windows users as well, albeit the current version of osquery-python needs to be ported.